src/background.ts

```diff
@@ -14,18 +13,23 @@ function getOauth2Client(origin: string) {
     ? env.DEV_OAUTH2_CLIENT_ID
     : env.OAUTH2_CLIENT_ID;
   const subjectId = isDev(origin)
-    ? env.DEV_OAUTH2_SUBJECT_ID
-    : env.OAUTH2_SUBJECT_ID;
+    ? null
+    : env.OAUTH2_SUBJECT_ID[env.renderBackend];
+  const subjectName = isDev(origin)
+    ? env.DEV_OAUTH2_SUBJECT_NAME[env.renderBackend]
+    : null;
   const clientAuthParams = [
     [
       'scope',
       `openid profile ${
         shopifyEmployee === true ? 'employee' : ''
-      } ${DEVTOOLS_SCOPE} ${COLLABORATORS_SCOPE}`,
+      } ${Object.values(env.DEVTOOLS_SCOPE).join(' ')} ${COLLABORATORS_SCOPE}`,
```

Comment:
Where is `DEVTOOLS_SCOPE` being defined now? Looks like it was removed in L4?

Reply:
It's a map now: https://github.com/Shopify/shopify-theme-inspector/pull/63/files#diff-9bcf0275d72a0ae3b1d14282a4a1a2dbR20.

One for each backend. Because Identity requires that we use a scope owned by the audience app (Core or SFR).